compute_sdk/tests/unit/test_client.py

```diff
@@ -331,6 +342,61 @@ def funk():
     assert func_data.metadata is None
 
 
+def test_register_function_no_function(gcc):
+    gcc.web_client = mock.MagicMock()
+
+    with pytest.raises(ValueError) as pyt_exc:
+        gcc.register_function(None)
+
+    assert "either" in str(pyt_exc).lower()
+    assert "'function'" in str(pyt_exc).lower()
+    assert "'function_name'" in str(pyt_exc).lower()
+    assert "'function_code'" in str(pyt_exc).lower()
+
+
+def test_function_registration_data_function():
+    frd = FunctionRegistrationData(function=funk)
+
+    assert frd.function_name == "funk"
+    assert frd.function_code == _get_packed_code(funk)
```

Comment:
Oh wow; not for this PR, but I don't think we have a test for `_get_packed_code`&nbsp;&mdash;&nbsp;unfortunate as that function is rather critical to this process.  Doh!